tests/unit/futures_test.cc

```diff
@@ -1259,9 +1255,6 @@ SEASTAR_TEST_CASE(test_futurize_from_tuple) {
     future<int> fut1 = futurize<int>::from_tuple(v1);
     future<> fut2 = futurize<void>::from_tuple(v2);
     BOOST_REQUIRE(fut1.get0() == std::get<0>(v1));
-#if SEASTAR_API_LEVEL < 5
-    BOOST_REQUIRE(fut2.get() == v2);
-#endif
```

Comment:
The `fut2` is not needed after this removal. So is `v2` thing up above that didn't fit into this hunk.

?